avocado/core/test.py

```diff
@@ -129,7 +129,8 @@ class Test(unittest.TestCase):
     default_params = {}
 
     def __init__(self, methodName='test', name=None, params=None,
-                 base_logdir=None, job=None, runner_queue=None):
+                 base_logdir=None, job=None, runner_queue=None,
+                 tests_tmpdir=None):
```

Comment:
Well I'd prefer not to change the test parameters as this would break all the custom test types and even user's custom test types (eg. avocado-vt, avocado-virt). We ought to add either `**kwargs` or custom args dictionary with extensible format for such changes in the next LTS... https://trello.com/c/9kz2ph6n/885-rework-avocado-test-init-arguments

Anyway for now I'd suggest using just one communication channel, therefor something like:

```
@property
def tests_tmpdir(self):
    return os.environ.get("XXX_TESTS_COMMON_TMPDIR")
```